KIPs/kip-37.md

```diff
@@ -0,0 +1,908 @@
+
+##### JSON Schema
+
+```json
+{
+  "title": "Token Metadata",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Identifies the asset to which this token represents"
+    },
+    "decimals": {
+      "type": "integer",
+      "description": "The number of decimal places that the token amount should display - e.g. 18, means to divide the token amount by 1000000000000000000 to get its user representation."
+    },
+    "description": {
+      "type": "string",
+      "description": "Describes the asset to which this token represents"
+    },
+    "image": {
+      "type": "string",
+      "description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive."
+    },
+    "properties": {
+      "type": "object",
+      "description": "Arbitrary properties. Values may be strings, numbers, object or arrays."
+    },
+    "localization": {
+      "type": "object",
+      "required": ["uri", "default", "locales"],
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "The URI pattern to fetch localized data from. This URI should contain the substring `{locale}` which will be replaced with the appropriate locale value before sending the request."
+        },
+        "default": {
+          "type": "string",
+          "description": "The locale of the default data within the base JSON"
+        },
+        "locales": {
+          "type": "array",
+          "description": "The list of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (http://cldr.unicode.org/)."
+        }
+      }
+    }
+  }
+}
+```
+
+##### Localized Sample
+
+Base URI:
+
+```json
+{
+  "name": "Advertising Space",
+  "description": "Each token represents a unique Ad space in the city.",
+  "localization": {
+    "uri": "ipfs://QmWS1VAdMD353A6SDk9wNyvkT14kyCiZrNDYAad4w1tKqT/{locale}.json",
+    "default": "en",
+    "locales": ["en", "es", "fr"]
+  }
+}
+```
+
+es.json:
+
+```json
+{
+  "name": "Espacio Publicitario",
+  "description": "Cada token representa un espacio publicitario único en la ciudad."
+}
+```
+
+fr.json:
+
+```json
+{
+  "name": "Espace Publicitaire",
+  "description": "Chaque jeton représente un espace publicitaire unique dans la ville."
+}
+```
+
+### Minting Extension
+
+The **minting extension** is OPTIONAL for KIP-37 smart contracts. This allows your contract to create and mint a new token.
```

Comment:
@kaikookim Please define the term "create" and "mint".

Reply:
OK